src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -854,6 +857,445 @@ void PerimeterGenerator::split_top_surfaces(const ExPolygons &orig_polygons, ExP
     //}
 }
 
+// find out if paths touch - at least one point of one path is within limit distance of second path
```

Comment:
Could we add credit to the original author?
Something like this:
// Port "extra perimeters on overhangs" from PrusaSlicer. Original author: PavelMikus pavel.mikus.mail@seznam.cz

Reply:
Done